Bottom: aff44b871980ab9e18c7d2c9b90bc34b060bb30d
Top:    69779eac4f1a8db7921abac748ff91b8fc330bae
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-09-10 14:35:09 -0700

Refresh of rs-docs

---

diff --git a/docs/rsocket b/docs/rsocket
new file mode 100644
index 0000000..87cc289
--- /dev/null
+++ b/docs/rsocket
@@ -0,0 +1,3 @@
+rsocket protocol and design guide
+
+
